docs/en/latest/plugins/error-log-logger.md

```diff
@@ -57,11 +65,10 @@ For more info on Batch-Processor in Apache APISIX please refer.
 | timeout                          | integer | optional    | 3                              | [1,...] | Timeout for the upstream to connect and send, unit: second.                                          |
 | keepalive                        | integer | optional    | 30                             | [1,...] | Time for keeping the cosocket alive, unit: second.                                                   |
 | level                            | string  | optional    | WARN                           |         | The filter's log level, default warn, choose the level in ["STDERR", "EMERG", "ALERT", "CRIT", "ERR", "ERROR", "WARN", "NOTICE", "INFO", "DEBUG"], the value ERR equals ERROR.         |
-| batch_max_size                   | integer | optional    | 1000                           | [1,...] | Max size of each batch.                                                                              |
-| inactive_timeout                 | integer | optional    | 3                              | [1,...] | Maximum age in seconds when the buffer will be flushed if inactive.                                  |
-| buffer_duration                  | integer | optional    | 60                             | [1,...] | Maximum age in seconds of the oldest entry in a batch before the batch must be processed.            |
-| max_retry_count                  | integer | optional    | 0                              | [0,...] | Maximum number of retries before removing from the processing pipe line.                             |
-| retry_delay                      | integer | optional    | 1                              | [0,...] | Number of seconds the process execution should be delayed if the execution fails.                    |
+
+### Attributes(Batch-processor)
```

Comment:
The doc doesn't make it clear what's batch processor attributes and why we need it in the logger...